.travis.yml

```diff
@@ -8,8 +8,10 @@ php:
   - 5.4
 
 install:
-  - composer install --dev --prefer-source
+  - composer install --prefer-source
 
 script:
   - bin/phpunit
   - bin/psecio-parse scan src tests
+
+dist: trusty
```

Comment:
That behavior is actually documented on Travis CI.

But I'd preffer changing dist only for hhvm builds like done in https://github.com/console-helpers/svn-buddy/blob/master/.travis.yml#L17